src/image.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2022 The peniko authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Also licensed under MIT license, at your choice.
+
+use super::{Blob, Extend};
+
+/// Defines the pixel format of an image.
+#[derive(Copy, Clone, PartialEq, Eq, Debug)]
+pub enum Format {
+    /// 32-bit RGBA with 8-bit channels.
```

Comment:
Really going all-out here supporting all the different formats, are we?

Reply:
I think this still fits nicely within the zero, one, many rule? :)